samples/dnn/action_recognition.py

```diff
@@ -0,0 +1,280 @@
+import os
+import copy
+import shutil
+import numpy as np
+import cv2 as cv
+import argparse
+
+parser = argparse.ArgumentParser(add_help=False)
+parser.add_argument('--input', help='Path to input video file.')
+parser.add_argument('--model', help='Path to model.')
+parser.add_argument('--classes', help='Path to classes list.')
+parser.add_argument('--temporal_unit', default=5, help='Averages the scores over temporal_unit x sample_duration clips.')
+parser.add_argument('--sample_duration', default=16, help='Number of frames for action detection.')
+
+# To get net download original repository https://github.com/kenshohara/video-classification-3d-cnn-pytorch.git
```

Comment:
Please remove `.git` at the end. `git clone https://github.com/kenshohara/video-classification-3d-cnn-pytorch` works too.